pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -167,6 +174,66 @@ func (og *operationGenerator) GenerateVolumesAreAttachedFunc(
 	}, nil
 }
 
+func (og *operationGenerator) GenerateBulkVolumeVerifyFunc(
```

Comment:
I am going to create some issues and if try and get saad/jing to assign them to myself.